components/sidebar/SidebarElement.vue

```diff
@@ -27,12 +27,12 @@ const logout = async () => {
 
 <template>
   <aside
-    class="flex max-h-screen min-w-[300px] flex-col border-r border-label-separator bg-background-2"
+    class="flex max-h-screen min-w-[300px] flex-col border-r border-label-separator bg-background-2 dark:bg-background-dark-2"
   >
     <div class="flex items-center p-3">
       <SiteLogo size="small" />
       <span
-        class="mx-2 mt-1 inline-block rounded-xl bg-tint px-[5px] text-[13px] leading-5"
+        class="mx-2 mt-1 inline-block rounded-xl bg-tint px-[5px] text-[13px] leading-5 text-black-1"
```

Comment:
No dark mode text color here?

Reply:
This text should always be dark, because the background-color is set to `tint`, which refers to a color on which white barely would be readable. I'll set it to black for `dark` as well - I guess that's more intuitive though more explicit and shows the intend for it to remain regardless of the mode.